new anti-matter feature, proper value inheritance from groups

jnweiger opened the pull request:
Prefix the height of an object with the letter 'A' to create anti-matter.
An anti-matter object is cut away (subtracted via difference() {} in scad) from all normal objects.

I've also added distribute scripts to generate a *.deb file for easier installation in Debian and Ubuntu Linux systems.

Commit:
added a dist rule.

Commit:
distribute scripts for generating a *.deb package added.

Commit:
Feature: antimatter. Cut away if height is prefixed with an A.

jnweiger committed:
Add files via upload

jnweiger committed:
Rename Medallie.svg to Inkscape-3D-Medallie.svg

jnweiger committed:
Rename cross.scad to Inkscape-3D-Medallie.scad

Commit:
raise feature: Offset along Z axis, to make cut-outs and balconies.
Inheriting auto values from enclosing groups.

Commit:
pep8 strikes again...

Commit:
Locally, pep8-1.5.7 and pep8-1.7.0 find nothing wrong with my indent. But the
travis check with pep8-1.5.7 still explodes. Let's try again.

Commit:
another overindented continuation line.....

l0b0 commented:
Thank you for the work! However, I'm afraid this fork of the project isn't maintained anymore. It would be unfair to the community to claim I've got this project covered when I'm no longer using it and busy with other stuff. It looks like your fork is the furthest along at the moment, would you maybe be willing to take over as the canonical repository holder? Also, I'll try to review the changes this time.

Commit:
preparing for 0.15        Fixed https://github.com/fablabnbg/inkscape-paths2openscad/issues/1#issuecomment-294257592
Line width of V7 code became a minimum line width,
rendering is now based on stroke-width
Refactored LengthWithUnit() from getLength()
Still missing: poly*_points[] refactoring in scad code - needed for V7 merge

jnweiger committed:
Update README.md

Commit:
0.15 Fixed https://github.com/fablabnbg/inkscape-paths2openscad/
     issues/1#issuecomment-294257592
     Line width of V7 code became a minimum line width,
     rendering is now based on stroke-width
     Refactored LengthWithUnit() from getLength()
     Finished merge with v7 code.
     Subpath in subpath are now handled very nicely.
     Added msg_extrude_by_hull_and_paths() outline mode with nested
     paths.

jnweiger commented:
Fine with me. I'll take over then. Thanks for starting a great inkscape extension.

jnweiger committed:
Update README.md

jnweiger committed:
Update README.md

jnweiger committed:
Update README.md

jnweiger committed:
Update README.md

jnweiger committed:
Update README.md

jnweiger committed:
Update README.md

Commit:
Expose scale feaure of openSCAD's extrude_linear() function for conical or pyramidal objects.

Commit:
upload url printed after make dist.

Commit:
pep8 uglification added.

Commit:
0.17 Release
Scale an object about its center, not about the openscad origin.
allow Scale: 60,90 %
Removed the name 'auto' from gui and code. Talk about extrusion and parsing.

Commit:
code mangling for pep8. I'll switch the 80char nonsense off, some day.

Commit:
Default for 'description parsing' should be true, of course! Sorry Thomas.

Commit:
Refactored the commands into a separate tab in the inx for a much cleaner user interface.
Added 'View in OpenSCAD' feature with pidfile for single instance.

Commit:
pep8 happiness is now a joy!

Commit:
Start of a translation attempt. Makefile added, and an exemplar german de.po file.

Commit:
A first generated german inx file. Hooray.
Still not perfect, as not all strings get picked up...

l0b0 commented:
Closing since I am no longer maintaining this fork. Please consider using a different fork as the authoritative one.
